galaxy/main/models/collection.py

```diff
@@ -68,6 +69,13 @@ def latest_version(self):
             collection=self
         ).latest('pk')
 
+    @property
+    def highest_version(self):
+        versions = CollectionVersion.objects.filter(collection=self)
+        pairs = [(v, semver.Version(v.version)) for v in versions]
+        pairs.sort(key=lambda x: x[1], reverse=True)
+        return pairs[0][0]
+
 
 class CollectionVersion(mixins.TimestampsMixin, pulp_models.Content):
```

Comment:
Could/should CollectionVersion be using a `semantic_version.django_fields.VersionField` for it's version field?
https://python-semanticversion.readthedocs.io/en/latest/django.html

Reply:
Please don't. `semantic_version` package is not compliant with the spec and hopefully gets replaced with better alternative soon.

Reply:
Which spec? And in what ways? I don't remember finding any ways `semantic_version` is not compliant with the semver spec/

Reply:
Collections spec.